x-team/auto-ui
# Fix keywords field width on tabular view

**Release Type:** *Breaking Changes* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2470

## Description

- Added a `max-width` to `keyword` container in tabular view



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Related PRs

*N/A*


## Steps to Test or Reproduce
1 - Go to the Admin page and choose a tabular view 
2 - In the `keyword` field, write multiple keywords until filling the whole field.
3 - The field should keep the same width. 
 


## Impacted Areas in Application

- Admin page in tabular view the field keyword



## Screenshots

![keyword_max_width](https://user-images.githubusercontent.com/5392202/48370114-be76c780-e68e-11e8-89a4-33d4764e6184.gif)
